chain-vote/src/committee.rs

```diff
@@ -119,6 +120,18 @@ impl MemberState {
     }
 }
 
+impl MemberSecretKey {
+    pub fn to_bytes(&self) -> [u8; 32] {
+        self.0.sk.to_bytes()
+    }
+}
+
+impl MemberPublicKey {
+    pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
It's generally better to provide a non-allocating `as_bytes()` accessor that returns a slice reference:

```rust
    pub fn as_bytes(&self) -> &[u8] {
        // ...
    }
```

Making an owned value is trivially composed: `key.as_bytes().to_vec()`

For an optimized moving conversion, an `impl From<MemberPublicKey> for Vec<u8>` could be added.

Reply:
The thing is that the underlying methods already return an owned array.

Reply:
Maybe it's a subject for a wider refactoring then. In `chain-network` and elsewhere we have `as_bytes`, make sense to keep the APIs similar.